pi_controller/cattown/src/cattown/__main__.py

```diff
@@ -0,0 +1,34 @@
+"""Command-line interface."""
+import click
+
+from .platform_mgr import PlatformManager
+from .cv import ComputerVisionSystem
+from .personalities.canned_pattern import DoesCannedPattern
+
+PERSONALITIES = [
+    # todo how does gcode work??? add some real gcode moves.
+    # (I see a text2gcodes() but not gcodes2text() in pygcode... does it only parse and not generate??????)
+    DoesCannedPattern([])
+]
+
+@click.command()
+@click.version_option()
+def main() -> None:
+    """Does Cat Town!!!!"""
+
+    platform_manager = PlatformManager()
+    platform_manager.initialize_hardware()
+
+    cv = ComputerVisionSystem()
```

Comment:
I'd prefer we don't call this `cv` because that's the common imported name of OpenCV and I've been following that convention so far.  Can we call it something like ccv, cat cv?

Reply:
Absolutely!!!